python/triton/knobs.py

```diff
@@ -503,6 +503,8 @@ class amd_knobs(base_knobs):
     use_buffer_ops: env_bool = env_bool("AMDGCN_USE_BUFFER_OPS", True)
     # Note: This requires use_buffer_ops be true to have any effect
     use_buffer_atomics: env_bool = env_bool("AMDGCN_USE_BUFFER_ATOMICS", True)
+    # Note: This requires use_buffer_ops be true to have any effect
+    use_buffer_small_tensor: env_bool = env_bool("AMDGCN_USE_BUFFER_ANALYZE_SMALL_TENSOR", True)
```

Comment:
analyze-small-tensor-ofst by default take false value, such that when the pass come across ld/st from small-tensor, it can blindly convert it into buffer-op.

Taking false value by default may sound counter-intuitive, however, if we were using word of negate-sense, that would be more confusing.